spec/dummy/config/application.rb

```diff
@@ -5,11 +5,13 @@
 %w[
   action_controller/railtie
   action_view/railtie
+  action_cable/engine
```

Comment:
👍

Reply:
New Rails/RSpec includes ActionCable tests and waits for it in example groups without checking if it used in the project. So we need to include it for test app in order not to get an exception. Maybe it wil be fixed in stable 4.0 release